app/controllers/admin/content_pages_controller.rb

```diff
@@ -92,6 +92,18 @@ def preview
       render json: { html: html }
     end
 
+    def preview_of_live
+      @page = ContentPage.new(title: @content_page.title,
```

Comment:
removing this part of the code removes the side navigation on the Content pages for some reason

Reply:
Yes, the navigation is managed by the @page.navigation? (I think) which is always true for content pages, but not for the footer pages.  It's strange and awkward, but was the only way I could get it to work for the way guy wrote it.

Reply:
The question of whether it needs to be cloned is another issue - again, to do with the way of doing his own versioning, rather than using the versions gem...